### PR TITLE
Add ordered lists number for 6 common load-balancing algorithms

### PR DESCRIPTION
**Pull request by regchiu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,11 +434,11 @@ The admin can specify the weight for each service. The ones with a higher weight
 4. Hash 
 This algorithm applies a hash function on the incoming requests’ IP or URL. The requests are routed to relevant instances based on the hash function result. 
 
-Dynamic Algorithms 
+5. Dynamic Algorithms
```
Comment (rockyooooooo) [inline README.md:437]: I believe this is another unordered list apart from `Static Algorithms`.

File: README.md [hunk 2]
```diff
@@ -434,11 +434,11 @@ The admin can specify the weight for each service. The ones with a higher weight
 4. Hash 
 This algorithm applies a hash function on the incoming requests’ IP or URL. The requests are routed to relevant instances based on the hash function result. 
 
-Dynamic Algorithms 
+5. Dynamic Algorithms 
 Least connections
```
Comment (rockyooooooo) [inline README.md:438]: This should be an ordered list, which belongs to `Dynamic Algorithms`.

Comment (rockyooooooo) [inline README.md:438]: This should be an ordered list, which belongs to `Dynamic Algorithms`.

File: README.md (hunk 1 above)
Comment (regchiu) [reply README.md]: Thank you.

**Pull request merged**